system/Commands/Generators/SessionMigrationGenerator.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Commands\Generators;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use CodeIgniter\CLI\GeneratorTrait;
+
+/**
+ * Generates a migration file for database sessions.
+ *
+ * @deprecated Use make:migration instead.
+ */
+class SessionMigrationGenerator extends BaseCommand
+{
+	use GeneratorTrait;
+
+	/**
+	 * The Command's group.
+	 *
+	 * @var string
+	 */
+	protected $group = 'Generators';
+
+	/**
+	 * The Command's name
+	 *
+	 * @var string
+	 */
+	protected $name = 'session:migration';
+
+	/**
+	 * the Command's short description
+	 *
+	 * @var string
+	 */
+	protected $description = '[DEPRECATED] Generates the migration file for database sessions, use "make:migration" instead.';
```

Comment:
The deprecation is uncalled for. Calling `session:migration` is much faster than calling `make:migration` plus options to make it compatible to the current result. Plus the options of `make:migration` are not compatible with `session:migration`. The latter does not have the options `--table` and `--dbgroup` so directing users to use that is a deadend.

Reply:
I have added option `-session` to `make:migration` to build `ci_session_migration_file` with two question from table name and database group with default values: 'ci_session', 'default'